lib/mj-single.js

```diff
@@ -622,23 +623,20 @@ function GetSVG(result) {
 //  Create the PNG file asynchronously, reporting errors.
 //
 function GetPNG(result) {
-  var svgfile = result.svgfile; delete result.svgfile;
   if (data.png) {
-    var batikCommands = ['-jar', BatikRasterizerPath, '-dpi', data.dpi, tmpfile + '.svg'];
+    var pngScale = data.scale;
+    var svgFile = result.svgfile; delete result.svgfile;
+    var width = result.width; delete result.width;
+    var height = result.height; delete result.height;
+    var check = function(err) {
```

Comment:
I prefer to see the `check` function *after* the `synch` function has been defined, but it is not actually an error.